operator/Makefile

```diff
@@ -80,7 +91,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=k3d-rancher-logging-explorer-registry:5000/${IMG}
```

Comment:
we can leave these for now but we should make an issue to discuss how local development should work

Reply:
will sort these out in next PR ? which suppose to revamp make file